### PR TITLE
Added footer

### PR DESCRIPTION
**Pull request by jilleJr**
> For Iver employees: WO0000000681896

Iver logotype as a footer on every page

Snagged logos from https://www.iver.com/. We had our logotypes in the intranet but only as PNGs.

## Preview

> Doesn't show well in GitHub, but in Docsify it renderes correctly: https://github.com/iver-wharf/iver-wharf.github.io/blob/6050ace/docs/_footer.md
>
> image

### CODE REVIEW

File: docs/_static/site.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+.iver-logo-left, .iver-logo-right {
+	line-height: 100px;
```
Comment (martin-linden) [inline docs/_static/site.css:3]: Should we use px or rem? just thinking since we brought this up on another MR.

**Review by iverestefans — DISMISSED** [review]
LGTM!

File: docs/_static/site.css (hunk 1 above)
Comment (jilleJr) [reply docs/_static/site.css:3]: Oh snap, caught red handed.

Yes, yes we should use `rem`. I'll fix

Comment (jilleJr) [reply docs/_static/site.css]: REM'ed it up

**Review by iverestefans — APPROVED** [review]
LGTM!

File: docs/_static/site.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+.iver-logo-symbol svg {
+	height: 3rem;
+}
+
+/* Utility classes 
+ *
+ * Example usage:
+ * ![my image](_images/some-image.png ':class=float-right')
+ *
+ * */
```
Comment (Pikabanga) [inline docs/_static/site.css:31]: Remove? :)

**Review by Pikabanga — DISMISSED** [review]

File: docs/_static/site.css (hunk 2 above)
Comment (jilleJr) [reply docs/_static/site.css:31]: Why?

Comment (Pikabanga) [reply docs/_static/site.css:31]: I skimmed it too fast. I thought it was just commented out CSS.

I think those classes are self explanatory imo though.

Comment (jilleJr) [reply docs/_static/site.css]: Well the usage is not self-explanatory, but that kind of documentation shouldn't live there anyway.

I ended up removing it as it's unused anyway

**Review by Pikabanga — DISMISSED** [review]

**Review by iverestefans — APPROVED** [review]
LGTM!

**Pull request merged**